src/App.vue

```diff
@@ -795,7 +792,7 @@ export default class App extends Mixins(
       // this is an IDIR user
       this.setUserPhone(userInfo.phone)
     } else {
-      console.log('Invalid user phone') // eslint-disable-line no-console
+      console.info('Invalid user phone') // eslint-disable-line no-console
```

Comment:
This displays the same as a console.log but it's a different logging level, which I believe to be more appropriate in this case.